README.md

```diff
@@ -30,18 +30,29 @@ Start a local authentik instance by following https://goauthentik.io/docs/instal
 Before starting the instance, add this static token to the `.env` file:
 
 ```
-AUTHENTIK_TOKEN=this-password-is-for-testing-dont-use
+AUTHENTIK_TOKEN=this-token-is-for-testing-dont-use
 ```
 
 Afterwards, tests can be run from your Editor or via CLI:
 
 ```
 export TF_ACC=1
 export AUTHENTIK_URL=http://localhost:9000
-export AUTHENTIK_TOKEN=this-password-is-for-testing-dont-use
-go test -timeout 30s ./... -count=1
+export AUTHENTIK_TOKEN=this-token-is-for-testing-dont-use
+go test -timeout 0 ./... -count=1
 ```
 
+If running from within the devcontainer, use the following instead with the correct URL:
```

Comment:
It probably makes sense to also add some basic instructions on how to use the devcontainer in the first place.

Reply:
Could do!
Wasn't sure if this should become essentially the only supported workflow or if the Readme should be split into local dev instructions vs devcontainer instructions.

Reply:
I'm not leaning either way. @tanberry thoughts?

Reply:
Hi @Gunsmithy, and thank you very much for this contribution. My thought is to make the instructions about the devcontainer as the primary instructions (seems more in line with concept of IaaS/Terraform), but also have secondary instructions (our existing ones?) for devs not using the devcontainer.

The structure could be simply a Head 2 (##) for each of the two sections, with clear titles such as `Run tests using a devcontainer` and `Run tests using local build` or whatever words make technical sense (mine might not lol).

For readers, it is nice to have fully separate procedurals based on their exact need, so I think separating these isa good call.

Not necessarily related to your contribution of the devcontainer, but I think this Readme needs some more info in general, up top, about how and why to use this repo. I guess experienced Terraformers will know, but it's good to help out newbies with some high-level conceptual intros. @Gunsmithy if you are interesting in adding that, great, would be appreciated, or we can do it later in a separate PR.

Reply:
Also, if either of you have ideas and suggestions on where within our regualr tech docs we cold link to this Readme, that would be great. This devcontainer seems something worth boasting about. :-)

Reply:
I'll probably just make your suggested changes for surfacing the Devcontainer instructions and moving local below, but we'll see where my changes take me!

As for where to reference this in the docs, a quick look would suggest a lack of the Terraform being referenced much at all.

As part of my evaluation on whether or not to use Authentik, one of the key points I looked for was whether or not Authentik had Terraform support. I however found this by simply Googling "Authentik Terraform" and was directed to the provider.

It might make sense to have another sub category under the Developer Setup docs, something like "Terraform Provider environment" to which instructions to the devcontainer could be linked.

Then of course once a Devcontainer is created for the frontend/backend components, those docs could be updated too.

I would tackle that one myself too, but the main Authentik codebase seems to be something of a mono-repo, which doesn't lend itself too well to devcontainers, though it could possibly done with some nesting and multiple container definitions in the repo per coding language or major component.

Reply:
Thanks @Gunsmithy for that suggestion! I like the idea of adding a topic called (as you suggest) "Terraform Provider environment". (Also agree about we need to make it more easily discoverable that we support TF.)

However, I am unsure if the point you make about our code-base being mostly a mono-repo is a major concern or somehow incompatible with claiming TF support. I'll defer to @rissson and @BeryJu on that call. :-)

Reply:
No issue with claiming Terraform support!

I just meant the mono-repo makes Dev Containers trickier, not Terraform, so implementing them and advertising as such may be more difficult.

Pushing Readme changes shortly. :)